src/main/java/organice/logic/commands/EditCommand.java

```diff
@@ -79,6 +72,23 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson));
     }
 
+    /**
+     * Returns a person that has the same NRIC with the NRIC specified.
+     *
+     * @param nric NRIC of the person to be searched.
+     * @param lastShownList List of persons.
+     */
+    private Person nricToPerson(Nric nric, List<Person> lastShownList) {
+        Iterator<Person> iterator = lastShownList.iterator();
+        while (iterator.hasNext()) {
+            Person person = iterator.next();
+            if (person.getNric().equals(nric)) {
+                return person;
+            }
+        }
+        return null;
```

Comment:
Instead of throwing null, should throw a `PersonNotFoundException`. Refer to the execute method of `MatchCommand.java` for an example.